build/templates/venv-commons

```diff
@@ -25,13 +25,33 @@ rocks_trees = {
 export LUAROCKS_CONFIG="$ROCKS_CONFIG"
 
 # duplicate package.[c]path even though we have set in resty-cli, so luajit and kong can consume
-export LUA_PATH="$ROCKS_ROOT/share/lua/5.1/?.lua;$ROCKS_ROOT/share/lua/5.1/?.ljbc;$ROCKS_ROOT/share/lua/5.1/?/init.lua;$ROCKS_ROOT/share/lua/5.1/?/init.ljbc;$KONG_VENV/openresty/site/lualib/?.lua;$KONG_VENV/openresty/site/lualib/?.ljbc;$KONG_VENV/openresty/site/lualib/?/init.lua;$KONG_VENV/openresty/site/lualib/?/init.ljbc;$KONG_VENV/openresty/lualib/?.lua;$KONG_VENV/openresty/lualib/?.ljbc;$KONG_VENV/openresty/lualib/?/init.lua;$KONG_VENV/openresty/lualib/?/init.ljbc;$KONG_VENV/openresty/luajit/share/luajit-2.1.0-beta3/?.lua;;"
-# support custom plugin
+export LUA_PATH="\
+$ROCKS_ROOT/share/lua/5.1/?.lua;$ROCKS_ROOT/share/lua/5.1/?.ljbc;\
+$ROCKS_ROOT/share/lua/5.1/?/init.lua;$ROCKS_ROOT/share/lua/5.1/?/init.ljbc;\
+$KONG_VENV/openresty/site/lualib/?.lua;$KONG_VENV/openresty/site/lualib/?.ljbc;\
+$KONG_VENV/openresty/site/lualib/?/init.lua;$KONG_VENV/openresty/site/lualib/?/init.ljbc;\
+$KONG_VENV/openresty/lualib/?.lua;$KONG_VENV/openresty/lualib/?.ljbc;\
+$KONG_VENV/openresty/lualib/?/init.lua;$KONG_VENV/openresty/lualib/?/init.ljbc;\
+$KONG_VENV/openresty/luajit/share/luajit-2.1.0-beta3/?.lua"
+# XXX EE
+if [ -d "./plugins-ee" ] ; then
+    plugins_ee="${workspace_path}/bazel-bin/build/ee/plugins-ee"
+    mkdir -p "$plugins_ee/kong/plugins"
+    for p in ./plugins-ee/*/; do
+        p_name=$(basename "$p")
+        p_target="${plugins_ee}/kong/plugins/${p_name}"
+        if ! [ -L "$p_target" ]; then
+            ln -s "$(realpath "${p}"/kong/plugins/"${p_name}")" "$p_target"
+        fi
```

Comment:
```suggestion
        ln -s "$(realpath "${p}"/kong/plugins/"${p_name}")" "$p_target"
```

Reply:
If user manually replace the symbolic link to a custom directory, should we override it?

Reply:
let's not consider that use case for now